Makefile

```diff
@@ -10,6 +10,9 @@ tx_url2 = https://www.transifex.com/api/2/project/edx-platform/resource/$(transi
 # This directory must match .babelrc .
 transifex_temp = ./temp/babel-plugin-react-intl
 
+test:
+	npm run test
+
```

Comment:
I'm seeing:
```
console.error
      Warning: unmountComponentAtNode(): The node you're attempting to unmount was rendered by React and is not a top-level container. Instead, have the parent component update its state and rerender in order to remove this component.
```
in the [build output](https://travis-ci.com/github/edx/frontend-app-library-authoring/builds/185000637). 

Is that something that you think is worth addressing in this PR?

If not, please just drop a note in Known Issues, maybe under its own sub-heading for tooling-related issues.

Reply:
I think it's an issue with Paragon's Modal. We don't use that function anywhere, and shortly after spitting out errors it complains specifically about the Modal component.

Reply:
Added to the README.